.github/workflows/ci-checks.yaml

```diff
@@ -25,28 +25,34 @@ jobs:
     runs-on: [default]
     steps:
       - uses: actions/checkout@v4
+
       - name: Use Node.js
         uses: actions/setup-node@v4
         with:
-          node-version: "20" # Specify the node version you need
+          node-version: "20"
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v3
+        with:
+          version: 8
 
-      - name: Cache npm
+      - name: Cache pnpm store
```

Comment:
Cache seems not to work. All dependent jobs are building again dependencies:
```
Cache not found for input keys: Linux-pnpm-33f61430d3061768f6253e524af76ebc0b69f0de9ff1b5aefe65c20dae028115, Linux-pnpm-
```
Let me first investigate why does happen now with the changes.

Reply:
new finding:
```
Warning: Path Validation Error: Path(s) specified in the action for caching do(es) not exist, hence no cache is being saved.
```